homeassistant/components/wolflink/manifest.json

```diff
@@ -1,10 +1,10 @@
 {
   "domain": "wolflink",
   "name": "Wolf SmartSet Service",
-  "codeowners": ["@adamkrol93"],
+  "codeowners": ["@adamkrol93", "@mtielen"],
```

Comment:
Do you intend to do more in this integration, otherwise it is not a requirement to add you here

Reply:
@jpbede  Yes as the current maintainer is not been reacting for 3 years and was talking to others how can improve this integration